library/Error.nucleus.js

```diff
@@ -27,6 +27,7 @@ class NucleusError extends Error {
    * Creates a Nucleus Error.
    *
    * @argument {String} errorMessage
+   * @argument {Object} options
```

Comment:
The options aren't mandatory...
`* @argument {Object} [options]`

We could also add this line, because it is a special value:

`* @argument {Error} [options.error]`